recipes/batch_tar/meta.yaml

```
@@ -0,0 +1,45 @@
+
+{% set name = "batch-tar" %}
+{% set name2 = "batch_tar" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name2 }}-{{ version }}.tar.gz
+  sha256: ac4b3764eaa6ffe837c4fe83c29c087d31c99ec0db4e4c6752659792b9485e4b
+
+build:
+  number: 0
+  skip: true  # [win]
```

Please, change it to 
```suggestion
  skip: true  # [win or py2k]
```

---

I think this can be made `noarch: python` instead?
```suggestion
  noarch: python
```

It's find it Windows builds fail for `noarch` packages